File: WebShop/src/app/order/Order.ts
export class Order {

    selected: boolean;
    position: number;
    orderId: number;
    invoiceId: string;
    custId: string;
    payed: boolean;
    orderDate: Date;
    totalPrice: number;
    shippingDate: Date;
    delivered: Date;
    status: number;

}

export class OrderLine {
    
    orderId: number;
    lineNo: number;
    prodId: string;
    item: string;
    qtyOrdered: number;
    price: number;
    canceled: boolean;
    status: number;

}

File: API/API/Orders/Order.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace API.Orders
{
    public class Order
    {
        public int OrderId { get; set; }
        public string InvoiceId { get; set; }
        public string CustId { get; set; }
        public bool Payed { get; set; }
        public DateTime OrderDate { get; set; }
        public decimal TotalPrice { get; set; }
        public DateTime ShippingDate { get; set; }
        public int Status { get; set; } // 1 = open, 99 = closed



    }
}
